testsuite/generate-format/tests/staged/network.yaml

```diff
@@ -131,7 +131,7 @@ ProtocolId:
     2:
       MempoolDirectSend: UNIT
     3:
-      StateSynchronizerDirectSend: UNIT
+      StateSyncDirectSend: UNIT
```

Comment:
just a reminder that this is not a breaking change since this is only used in BCS which doesn't care about labels.